test/ecs/ecs_metadata/ecs_metadata_test.go

```diff
@@ -57,25 +55,25 @@ func TestValidatingCloudWatchLogs(t *testing.T) {
 
 		end := time.Now()
 
-		ok, err := awsservice.ValidateLogs(logGroupName, LogStreamName, &start, &end, func(logs []string) bool {
```

Comment:
Having an integration test for ECS metadata which relies on the agent's own log getting uploaded to CWL is not very robust. Log messages can change at a developer's whim...
But if there are no better options then okay.